2024_lisbon.html

```diff
@@ -0,0 +1,146 @@
+                    data-target="#bs-example-navbar-collapse-1">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                <a class="navbar-brand page-scroll" href="index.html"><img src="img/about.png" class="navbar-img" />
+                    CoreDev Hacking</a>
+            </div>
+
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                <ul class="nav navbar-nav navbar-right">
+                    <li>
+                        <a class="page-scroll" href="index.html#about">About</a>
+                    </li>
+                    <li class="active">
+                        <a class="page-scroll" href="pastevents.html">Past Events</a>
+                    </li>
+                    <!-- <li>
+                        <a class="page-scroll" href="2022_atlanta.html">Next Event</a>
+                    </li> -->
+                    <li>
+                        <a class="page-scroll" href="faq.html">FAQ</a>
+                    </li>
+                    <li>
+                        <a class="page-scroll" href="index.html#contact">Contact</a>
+                    </li>
+                </ul>
+            </div>
+            <!-- /.navbar-collapse -->
+        </div>
+        <!-- /.container-fluid -->
+    </nav>
+    <section class="bg-dark">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+                    <h1 class="section-heading">October 2024 Lisbon</h1>
+                    <br />
+                    <h2>October 14-17 2024</h2>
+                    <p>CoreDev Meetup <em>(invite only)</em></p>
+                    <hr />
+                    Transcripts from the meetup are <a href="https://btctranscripts.com/bitcoin-core-dev-tech/2024-10/">available</a>.
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="bg-primary" id="info-milan">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+                    <h2 class="section-heading">Location</h2>
+                    <p>Lisbon, Portugal
+                    </p>
+                    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d99579.41434123636!2d-9.242136754648367!3d38.74405052698457!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0xd19331a61e4f33b%3A0x400ebbde49036d0!2sLisbon%2C%20Portugal!5e0!3m2!1sen!2sus!4v1732719113753!5m2!1sen!2sus" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
```

Comment:
is this necessary?

Reply:
We had it in on previous event and the page is already sparse, but I think its fine to leave out. Removed.